terraform/modules/gost_postgres/main.tf

```diff
@@ -45,15 +50,22 @@ module "db" {
   vpc_id                         = var.vpc_id
   subnets                        = var.subnet_ids
   create_security_group          = true
+  create_db_subnet_group         = true
   security_group_use_name_prefix = true
-  allowed_security_groups        = var.allowed_security_groups
-  allowed_cidr_blocks            = var.allowed_cidr_blocks
+  security_group_rules = {
+    for k, security_group_id in var.ingress_security_groups : k => {
+      type                     = "ingress"
+      source_security_group_id = security_group_id
+    }
+  }
```

Comment:
The [v7.x -> v8.x docs for this module](https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/blob/e054f77c700128c206b0912c51712313981d6fae/UPGRADE-8.0.md#security-group-rules-migration) includes a [section](https://github.com/terraform-aws-modules/terraform-aws-rds-aurora/blob/e054f77c700128c206b0912c51712313981d6fae/UPGRADE-8.0.md#security-group-rules-migration) for highlighting the caveats involved in this particular change. 

In my experience from testing in a sandbox environment, this actually caused `terraform apply` to fail, although the failure itself didn't cause any perceivable issues. While not ideal, the most straightforward way to deal with this is to let `terraform apply` fail initially, and then run `terraform apply` again- I didn't need to actually do any manual state manipulation (e.g. `terraform state mv ...`) to resolve things.